Update the documentation for machine annotations (#240)
dodona-edu/dodona-edu.github.io: en/guides/creating-a-judge/README.md

- An `Annotation` object consists of:

  - `row`, a zero-based index indicating the annotated line (start).

>   - An optional `column`, a zero-based index indicating the starting character to mark in the first `row`. When undefined all rows will be marked completely.

Reviewer: ```suggestion
  - An optional `column`, a zero-based index indicating the first character to mark in the first `row`. If undefined, all rows will be marked completely.
```

  - `text`, a string containing the annotation
  - `type`, the severity of the annotation, either `"info"`, `"warning"` or `"error"`.

>   - An optional `rows`, the amount rows that should be marked. This should be at least `1`. The default value is `1`.

Reviewer: ```suggestion
  - An optional `rows`, the number rows to mark. This should be at least `1`. The default value is `1`.
```

>   - An optional `columns`, the amount of characters to mark in the last `row`. The default value is `0`, which will put a marker before the `column` character.

Reviewer: ```suggestion
  - An optional `columns`, the number of characters to mark in the last `row`. The default value is `0`, which will put a marker before the `column` character.
```
